tests/integration/test_command_line_client.py

```diff
@@ -148,6 +149,26 @@ def test_command_line_client():
     assert_true(os.path.exists(downloaded_filename))
     assert_true(filecmp.cmp(filename, downloaded_filename))
 
+    # Store the same file and don't force a new version
+
+    # Get the existing file to determine it's current version
+    current_file = syn.get(file_entity_id, downloadFile=False)
+    current_version = current_file.versionNumber
+
+    # Store it without forcing version
+    output = run('synapse',
+                 '--skip-checks',
+                 'store',
+                 '--noForceVersion',
+                 '--id',
+                 file_entity_id,
+                 filename)
+
+    # Get the File again and check that the version did not change
+    new_file = syn.get(file_entity_id, downloadFile=False)
+    new_version = new_file.versionNumber
+    assert_equals(current_version, new_version)
+
```

Comment:
The version number should be bumped if the file uploaded is different.  Test that even if force_version is False that the version number is increased.

Reply:
I handled this by updating the unit tests for Entity.